nemoguardrails/library/autoguard/README.md

```diff
@@ -273,48 +273,44 @@ The colang file has to be in the following format:
 
 ```colang
 define flow call autoguard input
-  $input_result = execute autoguard_input_api
-  if $input_result["pii_fast"]["guarded"]
-    $pii_response_input = $input_result['pii_fast']['response']
-    bot respond pii input
-    stop
+  $input_result = execute autoguard_input_api(show_autoguard_message=True)
+
   if $input_result["guardrails_triggered"]
     $autoguard_input_response = $input_result['combined_response']
-    bot refuse to respond autoguard input
+    bot refuse to respond
     stop
 
 define flow call autoguard output
   $pre_rail_bot_message = $bot_message
-  $output_result = execute autoguard_output_api
-  if $output_result["pii_fast"]["guarded"]
-    $pii_response_output = $output_result['pii_fast']['response']
-    bot respond pii output
-    stop
+  $output_result = execute autoguard_output_api(show_autoguard_message=True)
+
   if $output_result["guardrails_triggered"]
-    $autoguard_output_response = $output_result['combined_response']
-    bot refuse to respond autoguard output
+    bot refuse to respond
     stop
   else
-    bot respond to question
+    $pii_message_output = $output_result["pii_fast"]["response"]
```

Comment:
same here, would this create a key error if pii_fast isn't enabled?

Reply:
I have kept a default pii response
https://github.com/decypher-ai/NeMo-Guardrails/blob/8a8f3767d83e1178961786b1a5eb109de37e0983/nemoguardrails/library/autoguard/actions.py#L87